Installing-and-Configuring/Prerequisites.md

```diff
@@ -60,6 +60,9 @@ USB Security Dongles (aka security token aka smartcard)
 
  *NOTE* - HOTP remote attestation is supported from Librem platforms by default, 
   Otherwise HOTP is explicitely supported by board configurations having `hotp` in their [board names](/Prerequisites#supported-devices).
```

Comment:
Hmm will have to review this, this is false for oem maintained boards but not related to this PR. Cood enough, merging.